src/Agent/Widget/Wrap.vue

```diff
@@ -18,7 +18,7 @@ export default {
   methods: {
     computedTime() {
       let time = 0
-      if(this.inqueue != undefined && this.inqueue.queue != undefined){
+      if(this.inqueue && this.inqueue.queue != undefined){
```

Comment:
Makes sense to remove undefined check from .queue too, no?

Reply:
yes